src/devices/RealtimeClock/RealtimeClock.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Device.Gpio" Version="$(SystemDeviceGpioPackageVersion)" />
```

Comment:
We recently changed this so that new .csprojs use:

```xml
    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj">
      <AdditionalProperties>$(AdditionalProperties);RuntimeIdentifier=linux</AdditionalProperties>
    </ProjectReference>
```

Can you please update this project, and your sample projects as well?

Reply:
Got it, but I hate this decision. Every time I need to put all `.csproj` in one `.sln`. Maybe there is something wrong with my VS settings.😅

Reply:
Oh that is because you have package restore enabled in your VS. Go to `Tools -> Options`  and then to `NuGet Package Manager -> General` and make sure that `Allow NuGet to download missing packages` and `Automatically check for missing packages during build in Visual Studio` are both **unchecked**. After doing this, all you will need for setup is to have at least run `dotnet restore` on the project once before opening VS (which we do when you call build.cmd) and then everything should just work fine without the need of pulling stuff into your solution.